src/main/java/appeng/core/features/registries/P2PTunnelRegistry.java

```diff
@@ -73,22 +73,12 @@ public void configure()
 		this.addNewAttunement( blocks.energyCell(), TunnelType.FE_POWER );
 		this.addNewAttunement( blocks.energyCellCreative(), TunnelType.FE_POWER );
 
-		/**
-		 * RF tunnel items
-		 */
-
-		this.addNewAttunement( this.getModItem( "thermaldynamics", "duct_0", 0 ), TunnelType.RF_POWER ); // leadstone
-																											// fluxduct
-		this.addNewAttunement( this.getModItem( "thermaldynamics", "duct_0", 1 ), TunnelType.RF_POWER ); // hardened
-																											// fluxduct
-		this.addNewAttunement( this.getModItem( "thermaldynamics", "duct_0", 2 ), TunnelType.RF_POWER ); // redstone
-																											// fluxduct
-		this.addNewAttunement( this.getModItem( "thermaldynamics", "duct_0", 3 ), TunnelType.RF_POWER ); // signalum
-																											// fluxduct
-		this.addNewAttunement( this.getModItem( "thermaldynamics", "duct_0", 4 ), TunnelType.RF_POWER ); // resonant
-																											// fluxduct
-		this.addNewAttunement( this.getModItem( "thermaldynamics", "duct_0", 5 ), TunnelType.RF_POWER ); // cryo-stabilized
-																											// fluxduct
+		this.addNewAttunement( this.getModItem( "thermaldynamics", "duct_0", 0 ), TunnelType.FE_POWER );
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "thermaldynamics" 15 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1192)